jsx/InstrumentForm.js

```diff
@@ -76,6 +77,15 @@ function renderRadio(element, key, onUpdate, isRequired) {
         hasError={isRequired && (!element.Value)}
         errorMessage="This field is required"
       />
+      <button className="asText" onClick={() => { var el = document.getElementsByName(element.Name);
+                                   for (var i=0; i < el.length; i++){
+                                     el[i].checked = false;
+                                   onUpdate(element.Name, null);}
+                                 }
+                          } type="button">
+        <span className="" aria-hidden="true"></span> Reset
+      </button> 
+      </div>
```

Comment:
Indentation is a bit off, the children of the top level `div` should be indented